internal/terraform/local.go

```diff
@@ -256,6 +258,8 @@ func Install(tfversion string, mirrorURL string) error {
 		mirrorURL = fmt.Sprintf("%s/", mirrorURL)
 	}
 
+	logrus.Debugf("downloading terraform version %s to: %s", tfversion, installLocation)
```

Comment:
I would add this line before we actually download. So we know which version we're trying to download. And when it fails we know what to look for. How does it sound?

Reply:
this line is already located before downloading

Reply:
probably @AutomationD means like info message before actual downloading